.github/workflows/pull-requests.yml

```diff
@@ -50,7 +55,17 @@ jobs:
         run: yarn
       - name: Run tests
         run: yarn test
-
+  
+  Formatting:
+    name: Check formatting
+    runs-on: ubuntu-latest
```

Comment:
1. Make this depend on `Linter`
1. Make `Continuous-Integration` depend on`Formatting`

That way we don't start the testing until the simpler tasks are complete. The way we have it now, formatting could fail while testing is occurring, but testing and other downstream activities will continue.

Reply:
Made the required changes in the latest commit.